Library/Homebrew/test/cask/cmd/uninstall_spec.rb

```diff
@@ -90,6 +90,31 @@
     expect(cask).not_to be_installed
   end
 
+  context "when Casks use script path with `~` as `HOME`" do
+    let(:dir) { TEST_TMPDIR }
+    let(:app) { Pathname.new("#{dir}/MyFancyApp.app") }
+    let(:cask) { Cask::CaskLoader.load(cask_path("with-uninstall-script-user-relative")) }
+
+    around do |example|
```

Comment:
```suggestion
    before do
      ENV["HOME"] = dir
    end
```

Reply:
Thanks. I was basing off of `cask/artifact/shared_examples/uninstall_zap.rb` and didn't notice `around` wasn't needed.

Changes added.